src/views/opm.tsx

```diff
@@ -628,23 +664,27 @@ const OPM: React.FC<OPMProps> = (props) => {
                               label={form.label}
                               optionLabel="name"
                               placeholder=""
-                              autoFocus={props.filters && (props.filters[form.name] !== undefined)}
+                              autoFocus={
+                                props.filters &&
+                                props.filters[form.name] !== undefined
```

Comment:
if it is undefined, it would have filtered out in the first check itself.
Why are u again checking for undefined ?